src/utils/formatDate.js

```diff
@@ -0,0 +1,9 @@
+export function formatDate(date) {
+  const day = String(date.getDate()).padStart(2, "0");
+  const month = String(date.getMonth() + 1).padStart(2, "0");
```

Comment:
Any specific reason to use padStart? I mean we can found day normally by using date.getDate(). Thanks!

Reply:
@iud-wtag vai I'm using `padStart` because it will format the day and month to always be of 2 letters. E.g. 1 will be converted to 01.

Reply:
What if that value of month is between `Oct` to `Dec` ? Thanks @nsi-wtag

Reply:
@azo-wtag vai it pads an extra 0 only for months and days having a single digit (1-9). As `Oct` to `Dec` are already double digit months, so it won't pad any 0, or won't manipulate their numbers.

![Screenshot 2023-06-25 at 2 27 23 PM](https://github.com/nsi-wtag/todo-app-react/assets/137053653/1d4ef02a-9900-404c-993f-c9ff3342476b)

![Screenshot 2023-06-25 at 2 28 39 PM](https://github.com/nsi-wtag/todo-app-react/assets/137053653/dab05a33-42ba-48d2-9fe8-b338ea678dfc)